From 40cea3efb4d2c2bd2d99827a76865edeb9da981d Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Thu, 30 Dec 2021 21:35:38 +0100
Subject: [PATCH] Remove call to ecm_find_qmlmodule(org.kde.kdeconnect)

This call can result in kdeconnectd being DBus-activated, which can cause unforseen side effects, such as the Windows CI breaking
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 649121b7..bf1439a0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,7 +38,6 @@ set_package_properties(KAccounts PROPERTIES
 )
 
 ecm_find_qmlmodule(Ubuntu.OnlineAccounts 0.1)
-ecm_find_qmlmodule(org.kde.kdeconnect 1.0)
 ecm_find_qmlmodule(org.kde.kquickcontrolsaddons 2.0)
 ecm_find_qmlmodule(org.kde.prison 1.0)
 
-- 
2.35.1

